sdk/test-utils/recorder/src/baseRecorder.ts

```diff
@@ -127,22 +122,23 @@ export class NockRecorder extends BaseRecorder {
   }
 
   public record(): void {
-    nock.recorder.rec({
-      dont_print: true
-    });
+    if (!isBrowser()) {
+      import("nock").then((nock) => {
```

Comment:
since import is async, I think record() should become an async function and then we can use await here.